examples_tf2_py/package.xml

```diff
@@ -15,7 +15,11 @@
   <author email="sloretz@openrobotics.org">Shane Loretz</author>
   <author email="tfoote@osrfoundation.org">Tully Foote</author>
 
+  <exec_depend>geometry_msgs</exec_depend>
   <exec_depend>launch_ros</exec_depend>
+  <exec_depend>rclpy</exec_depend>
+  <exec_depend>sensor_msgs</exec_depend>
+  <exec_depend>tf2_ros</exec_depend>
```

Comment:
Hm, this one shouldn't be necessary; this is strictly a python package, so the `tf2_ros_py` dependency should be enough.  The same goes for a few places below.